index.html

```diff
@@ -4,5 +4,25 @@
         <link rel="stylesheet" href="style/index.css">
     </head>
     <body>
+        <div style="width: 70%; height: 500px;">
```

Comment:
oops, forgot to mention -- since this is test code, let's not commit it to master. maybe throw it in the PR comment since it was used to generate the screenshot?

Reply:
I'm also down to have a `test.html` where we throw test / demo code for the modules.

FYI, there are some tools out there for exactly this, like this one: https://storybook.js.org/ (not suggesting that we use it... just something to be aware of)